### PR TITLE
Contact form section

### PR DESCRIPTION
**Pull request by ChrispaixK**
# Microverse_Portfolio : Contact Section

In this PR I and my partner :
* Add Contact section in html
* Add style for contact section in CSS
* Add an image for the Contact section background

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -219,13 +219,45 @@ <h3 class="li-1_title">languages <i class='fa fa-chevron-down'></i></h3>
...
+        <h2>Contact me</h2>
+        <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I'd love
+            to help with it
+        </p>
+        <form action="https://formspree.io/f/xbjwwqjp" method="post">
+            <ul class="contact-ul">
+                <li>
+                    <input type="text" class="input" name="name" placeholder="Enter your name" />
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly make sure the name input has _maxlength="30"_

File: index.html [hunk 2]
```diff
@@ -219,13 +219,45 @@ <h3 class="li-1_title">languages <i class='fa fa-chevron-down'></i></h3>
...
+                </li>
+                <li>
+                    <input type="email" class="input" placeholder="Enter your email" name="email" required />
+                </li>
+                <li>
+                    <textarea class="text-area" name="message" placeholder="Write your message here"
```
Comment (Kossi-stack) [inline index.html]: - [x] Kindly make sure the message input has _maxlength="500"_

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @ChrispaixK,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @ChrispaixK,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ChrispaixK** [comment]
Thank you @TedLivist  and @Kossi-stack 👍🏾

**Pull request merged**